webpack.prod.config.ts

```diff
@@ -19,6 +19,8 @@
 var commonWebpackConfig = require('./webpack.config.ts');
```

Comment:
Did you test this PR in both dev and prod environments?

Reply:
yes tested it prod env, build as expected Also tested via `start.sh --prod-env` to check the webpack build you have more suggestions?

Reply:
Done Added here #7224 Thanks!